aptos-move/framework/move-stdlib/sources/bcs.move

```diff
@@ -3,9 +3,14 @@
 /// published on-chain. See https://github.com/aptos-labs/bcs#binary-canonical-serialization-bcs for more
 /// details on BCS.
 module std::bcs {
-    /// Return the binary representation of `v` in BCS (Binary Canonical Serialization) format
+    /// Returns the binary representation of `v` in BCS (Binary Canonical Serialization) format.
+    /// Returns  `0x1C5` if serialization fails.
     native public fun to_bytes<MoveValue>(v: &MoveValue): vector<u8>;
 
+    /// Returns the size of the binary representation of `v` in BCS (Binary Canonical Serialization) format.
+    /// Returns  `0x1C5` if there is a failure when calculating serialized size.
```

Comment:
you mean aborts with that error code, right?

Reply:
Yes, I cannot add a constant because then the error message appears in output, which might change some behaviour. Having a the constant specified here at least will help to understand errors better.

Reply:
update the text. I would read it as if I need to check that return value (size) is not 0x1C5 itself

Reply:
Ah, true, good point.

Reply:
done